UKHomeOffice/sgmr-service
# SPCR-150 Update content on Landing (root) page

## Description
Content needs to be changed to be more useful to the user. Update the content to match the attached image.

<img width="687" alt="Screenshot 2022-01-07 at 15 00 02" src="https://user-images.githubusercontent.com/72464825/149487411-d6e62a3e-1fe4-42b2-b5c2-07f2351aef23.png">

### To test

- Start the application from scratch
- This should open on the landing page
- (if not - click on the gov logo)
- Check the content matches the image

## Developer Checklist

\* Required






Ensure you delete the branch once the Pull Request is merged.
